clar2wasm/src/words/consensus_buff.rs

```diff
@@ -71,9 +74,78 @@ impl Word for ToConsensusBuf {
     }
 }
 
+#[derive(Debug)]
+pub struct From;
```

Comment:
This is a name collision with a std lib trait and should be spelled out in full.

Reply:
Fair, I will expand this.